lib/page/PostHos.dart

```diff
@@ -20,18 +20,6 @@ class _PostHosPageState extends State<PostHosPage> {
         backgroundColor: Color(0xFFC37447),
       ),
       body: PostHosTable(),
-      // body: Padding(
```

Comment:
File name should start with lowercase character (Camel case ) . Folder name as well.